### ISSUE TITLE
prettyclass should be marked deprecated in favor of pretty

### ISSUE THREAD

**Issue by ddssff** [haskell/pretty#20]
http://hackage.haskell.org/package/prettyclass-1.0.0.0

**Comment by dterei** [comment]
Yeah, I've asked Lennart to do this and he hasn't responded yet. I'll bug him again.

**Comment by ddssff** [comment]
ping

**Comment by dterei** [comment]
I've contacted Lennart a few times and he either doesn't respond, or responds saying he'll do it, and then nothing happens. I've given up at this point unless I bump into him in person.

**Comment by ddssff** [comment]
Yeah, I run into this problem all the time.  It is troubling.
